src/npm-fastui/src/components/FormField.tsx

```diff
@@ -304,6 +305,62 @@ export const FormFieldSelectSearchComp: FC<FormFieldSelectSearchProps> = (props)
   )
 }
 
+interface FormFieldRadioProps extends FormFieldRadio {
+  onChange?: PrivateOnChange
+}
+
+export const FormFieldRadioComp: FC<FormFieldRadioProps> = (props) => {
+  const { name, required, locked, options, initial } = props
+  const className = useClassName(props)
+  const inputClassName = useClassName(props, { el: 'radio-option' })
+
+  return (
+    <div className={className}>
+      <Label {...props} />
+      {options.map((option, i) => {
+        if ('options' in option) {
+          // option is a SelectGroup
+          return option.options.map((subOption, j) => (
```

Comment:
probably nicer if you use the same logic we have for `SelectOptionComp` where we have one component that can call itself recursively to avoid duplication.